### ISSUE TITLE
alt+enter shortcut closes editor and goes to next row

### ISSUE THREAD

**Issue by neerup** [TatuLund/GridFastNavigation#102]
Hi.. 
Is it possible to have the plugin, check for ctrl/alt/shift when checking for enter key in open editor?
We use alt+enter shortcut as an select and return value all over in our project, but when in an open editor in a grid, the alt+enter shortcut also triggers a close editor and go to next row, resulting in next row/cells value being returned...
Could Keys.java check for ctrl/al/shift ?
From
`
public static boolean Keys.isRowChangeKey(key){return rowChangeKeys.contains(keyCode);};
`
to
`
public static boolean Keys.isRowChangeKey(event) {
 return rowChangeKeys.contains(event.getKeyCode()) 
         && !event.getCtrlKey()  && !event.getAltKey() && !event.getShiftKey();
}
`

**Comment by TatuLund** [comment]
I think this would be trivial change to do.

**Comment by neerup** [comment]
> I think this would be trivial change to do.

HI...
I think You are absolutely right.... 
I decided I couldn't wait :) So I went ahead and and made one tiny change to the code at it fixed it for me... But a fixed version here would be very appreciated.
Here is my tiny fix.. 
```
@@ -210,7 +220,7 @@ public class EditorStateManager {
                     move = true;
                 }
                 
-                if (validationError == -1 && Keys.isRowChangeKey(key)) {
+                if (validationError == -1 && Keys.isRowChangeKey(key) && !e.getAltKey() && !(e.getShiftKey() && e.getCtrlKey()) && !e.getCtrlKey()) {
                        saveContent();
```

**Comment by TatuLund** [comment]
Included in 2.6.0
